modules/Link.js

```diff
@@ -22,6 +22,10 @@ function isEmptyObject(object) {
   return true
 }
 
+function resolveToLocation(to, router) {
```

Comment:
I think this just ends up being the cleanest way to strip off the `to` prop before it gets to the underlying `<a>`.